tests/mr/device/skip_async.hpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <rmm/detail/error.hpp>
+
+#include <cuda_runtime_api.h>
+
+namespace rmm::test {
+
+/**
+ * @brief Should tests be skipped if cudaMallocAsync is not supported with the CUDA driver/runtime
+ * version?
+ */
+inline bool should_skip_async()
+{
+  static auto runtime_version{[] {
+    int runtime_version{};
+    RMM_CUDA_TRY(cudaRuntimeGetVersion(&runtime_version));
```

Comment:
I think this should be a static member of the `cuda_async_memory_resource`. And it should use the simple check that is already in the ctor, not check explicit driver and runtime versions.  The check in the ctor already factors in both driver and runtime.

Reply:
Done.